### PR TITLE
Delete get ping api

### LINKED ISSUE #8: Ping API

### PR DESCRIPTION
**Pull request by SOPHIA2401**
### Description
Commits performed for 
1. Delete Index API
2. Get Document API
3. Ping API

### Issues Resolved
#8, #9 && #10 resolved
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: model/_global/ping/operations.smithy [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+namespace OpenSearch
+
+@readonly
+@suppress(["HttpUriConflict"])
+@http(method: "GET", uri: "/")
+@documentation("It checks whether the cluster is up and available to process requests.")
```
Comment (sachetalva) [inline model/_global/ping/operations.smithy:12]: Can we change this to the description used here: https://github.com/opensearch-project/OpenSearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/api/ping.json

File: model/indices/delete/structures.smithy [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+}
+
+structure DeleteIndexOutput {
+
+    @required
+    Index: IndexName,
+
+    acknowledged:Boolean
+}
```
Comment (sachetalva) [inline model/indices/delete/structures.smithy:34]: np: missing newline at end of file(pls check all files for missing newlines)

File: .smithy.lsp.log [hunk 3]
```diff
@@ -0,0 +1,368 @@
+[09:57:59] [09:57:58] Created a temporary folder for file contents /var/folders/nn/63tk2_d96mzf32qwypk1l7km0000gs/T/smithy-lsp14316781329401607384
```
Comment (sachetalva) [inline .smithy.lsp.log:1]: was this file added intentionally?

Comment (SOPHIA2401) [reply .smithy.lsp.log:1]: Please Ignore this file, added unintentionally.

File: model/common_datatypes.smithy [hunk 4]
```diff
@@ -8,6 +8,7 @@ namespace OpenSearch
 
 @pattern("^[^+_\\-\\.][^\\\\, /*?\"<>| ,#\\nA-Z]+$")
 string IndexName
+string IdName
```
Comment (pgtgrly) [inline model/common_datatypes.smithy:11]: please add a newline between parameters

File: model/indices/get_document/structures.smithy [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+structure GetDocumentSourceInput {
+    @httpLabel
+    @required
+    index: IndexName,
+
+    @httpLabel
+    @required
+    id: IdName,        
+
+    // Common options to be removed by mixins start
```
Comment (pgtgrly) [inline model/indices/get_document/structures.smithy:87]: Please see how common options are named in the existing files

File: model/common_datatypes.smithy (hunk 4 above)
Comment (SOPHIA2401) [reply model/common_datatypes.smithy:12]: done

**Review by pgtgrly — APPROVED** [review]

**Review by sachetalva — APPROVED** [review]

**Pull request merged**